stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -48,6 +50,24 @@
   private final InspectModeHandler mInspectModeHandler;
   private Listener mListener;
 
+  // We don't yet have an an implementation for reliably detecting fine-grained changes in the
+  // View tree. So, for now at least, we have a timer that runs every so often and just reports
+  // that we changed. Our listener will then read the entire DOM from us and transmit the changes to
+  // Chrome. Detecting, reporting, and traversing fine-grained changes is a future work item.
+  private boolean mIsReportChangesTimerPosted = false;
+  private final Runnable mReportChangesTimer = new Runnable() {
+    @Override
+    public void run() {
+      mIsReportChangesTimerPosted = false;
+
+      if (mListener != null) {
+        mListener.onChanged();
```

Comment:
This is what computes the diff?  The semantics read very strangely to me, as if this code has already detected a change and is telling something about it.

Reworking the type/variable name of `mListener` and possibly modifying `onChanged` to be `onPossibleChange` might clear this up